src/app/login-dialog/login-dialog.component.scss

```diff
@@ -0,0 +1,167 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/*.login-bg:before {
+  content: "";
+  position: fixed;
+  left: 20%;
+  right: 0;
+  width: 60%;
+  height: 60%;
+  z-index: -1;
+
+  display: block;
+  background-image: url('/assets/images/background.png');
+
+  filter: blur(3px) brightness(80%);
+}*/
+
+.login-bg {
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  top: 0;
+  z-index: 0;
+}
+
+.social-logo {
+  width: 30px;
+  height: 30px;
+  padding-right: 10px;
+}
+
+app-login {
+  display: flex;
+  flex: 1;
+}
+
+.social-button {
+  margin-bottom: 5px;
+}
+
+.github {
+  background-color: #24292E;
+  color: white;
+}
+
+.google {
+  background-color: #D9453D;
+  color: white;
+}
+
+.twitter {
+  background-color : rgb(66, 156, 214);
+  color: white;
+}
+
+mat-card {
+  padding: 8px;
+}
+
+mat-card-content {
+  padding: 8px;
+}
+
+mat-icon {
+  margin: 5px;
+}
+
+.vertical-divider {
+  border-left: 1px solid rgba(0, 0, 0, 0.12);
+  width: 5px;
+  margin: 0 5px;
+}
+
+.close-modal{
+  cursor: pointer;
+}
+
+.error {
+  color: red;
+  margin-left: 4%;
+}
+
+
+
+
+.loader {
+  margin: 100px auto;
+  font-size: 15px;
+  width: 1em;
+  height: 1em;
+  border-radius: 50%;
+  position: relative;
+  text-indent: -9999em;
+  -webkit-animation: load5 1.1s infinite ease;
+  animation: load5 1.1s infinite ease;
+  -webkit-transform: translateZ(0);
+  -ms-transform: translateZ(0);
+  transform: translateZ(0);
+}
+@-webkit-keyframes load5 {
```

Comment:
What is this? Do they belong to the loader? I'd look for a loader component from angular material